### PR TITLE
added uninstall instructions

### LINKED ISSUE #121: create a guide for the Notation cleanup steps
We need to create a guide for the Notation cleanup steps. I think it will be used in multiple docs, we can easily reference a common article in different docs without maintaining duplicated content.

_Originally posted by @FeynmanZhou in https://github.com/notaryproject/notaryproject.dev/pull/114#discussion_r1050936879_

### PR DESCRIPTION
**Pull request by zr-msft**
Fixes #121

### CODE REVIEW

File: content/en/docs/installation/uninstall.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Uninstall the notation CLI and its configuration
+description: Uninstall the notation CLI, plugins, and configuration files
+weight: 2
+---
+
+The `notation` CLI uses the following directories for its binaries and configuration files:
+
+- `NOTATION_BIN` is the directory for the `notation` binary.
```
Comment (toddysm) [inline content/en/docs/installation/uninstall.md:9]: Are those env variables? They look like such... Shouldn't by convention they use $ in documentation? Are they automatically set during the installation? We should clarify that.

File: content/en/docs/installation/uninstall.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Delete the `NOTATION_BIN` directory to remove the `notation` binary.
+
+## Remove the additional binaries
+
+Delete the `NOTATION_LIBEXEC` directory to remove the additional binaries, including plugins.
+
+## Remove the configuration files
+
+**IMPORTANT:** `notation` automatically creates the `NOTATION_CONFIG` directory if it does not exist. If you remove the `NOTATION_CONFIG` directory, `notation` will recreate it the next time you run a command.
```
Comment (toddysm) [inline content/en/docs/installation/uninstall.md:25]: This is confusing as experience. It seems the first command someone runs creates this directory but I assume it doesn't set any env variables to make the location known - correct? I think we should clarify that _any_ `notation` command does that.

File: content/en/docs/installation/uninstall.md (hunk 1 above)
Comment (zr-msft) [reply content/en/docs/installation/uninstall.md:9]: Based on my testing, you can set these as environment variables to update the default location, but they are not set by default. `notation` seems to use a default value per platform if no value is present.

Updated the IMPORTANT callout to reflect this.

File: content/en/docs/installation/uninstall.md (hunk 2 above)
Comment (zr-msft) [reply content/en/docs/installation/uninstall.md]: updated.

File: content/en/docs/installation/uninstall.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+weight: 2
+---
+
+The `notation` CLI uses the following directories for its binaries and configuration files:
+
+- `NOTATION_BIN` is the directory for the `notation` binary.
+- `NOTATION_LIBEXEC` is the directory for other binaries, such as plugins, used by the `notation`.
+- `NOTATION_CONFIG` is the directory for the configuration files, including trust stores and trust policies.
+
+**IMPORTANT:** `notation` has default values for `NOTATION_BIN`, `NOTATION_LIBEXEC`, or `NOTATION_CONFIG`, but `notation` does not set those environment variables. If you wish to override the default values, you must set the environment variables. For more details on each directory as well as the default location of those directories on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/how-to/directory-structure" >}}).
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/docs/installation/uninstall.md]: adjusted the language to incorporate your feedback.

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Review by iamsamirzon — APPROVED** [review]

**Pull request merged**